### DISCUSSION TITLE
Drawio plugin making doc?

### DISCUSSION THREAD

**Discussion by I2rys** [jgraph/drawio#2703] [Q&A]
Does Drawio have a doc for It's plugin functions & etc? Because I'm having a hard time making a plugin plus Diagrams.net does not seem to have a plugin reload button.

**Comment by OlegZharkov** [comment]
> Does Drawio have a doc for It's plugin functions & etc? Because I'm having a hard time making a plugin plus Diagrams.net does not seem to have a plugin reload button.

I have the same struggle, some docs would be extremly useful, otherwise we do reverse engeenering.
